scipy/spatial/distance.py

```diff
@@ -1050,7 +1049,7 @@ def pdist(X, metric='euclidean', p=2, w=None, V=None, VI=None):
        Computes the Jaccard distance between the points. Given two
        vectors, ``u`` and ``v``, the Jaccard distance is the
        proportion of those elements ``u[i]`` and ``v[i]`` that
-       disagree where at least one of them is non-zero.
+       disagree.
```

Comment:
I believe this was redundant.  If two elements disagree, it goes without saying that at least one of them is non-zero.  If that weren't true, then they'd both be zero and would agree.